src/test/java/uk/gov/hmcts/reform/sscs/service/LetterUtilsTest.java

```diff
@@ -62,6 +52,31 @@
 public class LetterUtilsTest {
     private static final Subscription EMPTY_SUBSCRIPTION = Subscription.builder().build();
 
+    static Appellant APPELLANT_WITHOUT_ID = Appellant.builder()
+            .name(Name.builder().firstName("Tom").lastName("Cat").build())
+            .address(Address.builder().line1("Appellant Line 1").town("Appellant Town").county("Appellant County").postcode("AP9 3LL").build())
+            .build();
+
+    static  Representative REPRESENTATIVE_WITH_ID = Representative.builder()
+            .id("REP123456")
+            .name(Name.builder()
+                    .title("Mr.")
+                    .firstName("Representative")
+                    .lastName("Appellant")
+                    .build())
+            .build();
+    static Appellant APPELLANT_WITH_ID = Appellant.builder()
+            .id("APP123456")
+            .name(Name.builder().firstName("Tom").lastName("Cat").build())
+            .address(Address.builder().line1("Appellant Line 1").town("Appellant Town").county("Appellant County").postcode("AP9 3LL").build())
+            .build();
+
+    static JointParty JOINT_PARTY_WITH_ID = JointParty.builder()
+            .id("JP123456")
+            .name(Name.builder().firstName("Joint").lastName("Party").build())
+            .address(Address.builder().line1("Appellant Line 1").town("Appellant Town").county("Appellant County").postcode("AP9 3LL").build())
+            .build();
```

Comment:
This is quite lengthy for static variables. I'd recommend changing them to non-static and initialise in a BeforeEach.

Reply:
Removed and moved into setup method.